vector/src/main/java/im/vector/app/features/home/room/detail/composer/rainbow/RainbowGenerator.kt

```diff
@@ -29,61 +31,63 @@ class RainbowGenerator @Inject constructor() {
 
     fun generate(text: String): String {
         val split = text.splitEmoji()
-        val frequency = 360f / split.size
+        val frequency = 2 * Math.PI / split.size
 
         return split
                 .mapIndexed { idx, letter ->
                     // Do better than React-Sdk: Avoid adding font color for spaces
```

Comment:
I forgot to include it in the PR but you can remove the first half of this comment if you want.

Reply:
I see, will do! Thanks